app/models/assignment.rb

```diff
@@ -894,6 +898,19 @@ def add_graders_to_criterion(criterion_name, graders)
 
     criterion.add_tas_by_user_name_array(graders)
   end
+
+  # Returns the groupings of this assignment associated with the given section
+  def section_groupings(section)
+    section_groupings = Array.new
```

Comment:
Yes, I noticed that after I committed/pushed. Removing that line is in my next commit. I just haven't pushed yet.